### PR TITLE
feat(src): implement static formatting toolbar - I35

### PR DESCRIPTION
**Pull request by jolanglinais**
# Issue #35 

### Changes
- Adjust linting
- Install icons
  - Move into svg component
  - Include specifications
- Implement static toolbar
  - Utilize portal
  - Render custom icons
  - Adjust styling
  - Optimize event handler with pointer events
- Adjust plugin icon styling
  - Color change on active state
- Create grid styling
- Remove hover component
- Adjust schema
  - Implement underline case
  - Add code support for lists
- Modularize method functionality
- List functionality
  - Handle list blocks to and from

### Flags
- ~Researching `FormatToolbar` functionality~
- ~Needs all button functionality corrected~
- ~Needs color change on active (already bolded, for example)~
- Needs parameter functionality
- Needs fix for clicking list button on a list item twice.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -195,28 +195,16 @@ function MarkdownEditor(props) {
...
-            // it doesn't matter where we put the annotation
-            // so we use the current selection
-            const annotation = {
-              key: `lockText-${uuidv4()}`,
-              type: 'lockText',
-              anchor: selection.anchor,
-              focus: selection.focus,
-            };
-            editor.addAnnotation(annotation);
+    if (props.lockText && editorRef && editorRef.current) {
```
Comment (jeromesimeon) [inline src/index.js]: This seems to be picking the wrong version? Shouldn't the version from master be kept?

File: src/index.js [hunk 2]
```diff
@@ -394,8 +388,13 @@ function MarkdownEditor(props) {
   * Returns true if the selection is inside a variable
   * @param {*} value the Slate editor value
   */
-  const isInVariable = useCallback(value => isInVariableEx(value, value.selection.anchor), [isInVariableEx]);
-
+  const isInVariable = ((value) => {
```
Comment (jeromesimeon) [inline src/index.js]: Same comment as above, shouldn't we keep the version from master?

File: src/markdown/toMarkdown.js [hunk 3]
```diff
@@ -3,9 +3,12 @@ import Markdown from './Markdown';
 
 const NL = '\n';
 
+<<<<<<< HEAD
```
Comment (jeromesimeon) [inline src/markdown/toMarkdown.js]: This file wasn't merged.

**Review by jeromesimeon — CHANGES_REQUESTED** [review]
Replaced by #42 so should be closed.

**Comment by irmerk** [comment]
Closed in favor of #42

**Pull request closed without merging**